cmd/falco/snippet.go

```diff
@@ -142,18 +135,20 @@ func (s *Snippet) fetchAccessControl(c _context.Context, version int64) ([]snipp
 	return snippets, nil
 }
 
-func (s *Snippet) fetchBackend(c _context.Context, version int64) ([]snippetItem, error) {
-	backends, err := s.client.ListBackends(c, version)
+func (s *Snippet) fetchBackend() ([]snippetItem, error) {
+	var snippets []snippetItem
+	backends, err := s.fetcher.Backends()
 	if err != nil {
-		return nil, fmt.Errorf("failed to get backends for version %w", err)
+		return nil, fmt.Errorf("Failed to get Backends: %w", err)
+	}
+	if len(backends) == 0 {
```

Comment:
**question**: was the Fastly API remote code failing previously if there weren't any backends in your Fastly service, or is this an optimisation?

Reply:
it failed with if there are no backends so I resolved it by returning early

Reply:
ah thanks for fixing this then, sorry I missed it in my PR!